### DISCUSSION TITLE
Would it be possible to add more sortable columns?

### DISCUSSION THREAD

**Discussion by h3xpl01t** [collinbarrett/FilterLists#4192] [Ideas]
Just wanted to say thanks for all your hard work! I use the site all the time when I'm setting up a new PC. But I have an issue / suggestion. Much of the data I want to know about a blocklist is not visible / sortable while searching. I wanted to know if it could be added in a future update.

For example, it'd be super useful to be able to know and sort blocklists by: 

- Date Created
- Dated Modified
- File Size
- Number of Entries (or Lines)
- Approximate Number of Downloads

Knowing the Date Created / Date Modified can help a user determine if a list is actively being developed, or if there's an alternative or fork that would be better to use. The File Size / Number of Entries would be useful for determining how comprehensive a blocklist is (and for people stuck using Manifest v3 it could be a deciding factor, due to its limitations). And knowing the Number of Downloads would help newer users find the most useful lists.

I'm not sure if some of that data may be unavailable (or difficult to scrape), and I also understand that having too many columns could be distracting for some users. But if it is possible to pull the data from the site hosting the list, would you consider adding that as an option? And if it leads to the list looking too chaotic, maybe have a button or a menu to change which columns are visible?

Thank you!

**Comment by collinbarrett** [comment]
Heard.

I love these ideas, and I have wanted to implement some of them for some time. It's just a matter of finding development time.

Thanks.

**Comment by hiccups1980** [comment]
I miss that too. Because some lists are totally abandoned. And than you add them and you see "updated 1873" (you know what I mean).
